content/manager/update-deployment.md

```diff
@@ -6,58 +6,70 @@ draft: false
 
 ### Describing a Deployment Update
-The contents of the deployment update should be described in a [yaml blueprint file](http://docs.getcloudify.org/3.4.0/blueprints/overview/), just as any application in Cloudify. Following the aforementioned example, The updated application blueprint will probably include a new database type, a few new node templates of the new database type, and a few new relationships representing how these new nodes are connected to the existing architecture.
+The contents of the deployment update should be described in a [yaml blueprint file](http://docs.getcloudify.org/3.4.0/blueprints/overview/), just as any application in Cloudify. Following the aforementioned example, The updated application blueprint will probably include a new database type, a few new node templates of the new database type, and a few new relationships representing how these new nodes should be connected to the existing architecture.
 
 ### Using the CLI to Update a Deployment
 One quick way to update your deployment with Cloudify is using the CLI. Another way, perhaps more 'visual' is using the Cloudify UI. Updating a deployment via the CLI is quite reminiscent of uploading a blueprint or creating a deployment. You'll need a blueprint file describing your deployment update. That blueprint can be uploaded directly by supplying a local file path, or it can be upload as an archive.
-##### via a blueprint file
+#### via a blueprint file
 ```shell
 cfy deployments update -d ID_OF_DEPLOYMENT_TO_UPDATE -p PATH_TO_BLUEPRINT
 ```
 When updating a deployment using a blueprint file, the directory containing the blueprint file is packaged and uploaded as a whole.
-##### via an archived blueprint
+#### via an archived blueprint
 ```shell
 cfy deployments update -d ID_OF_DEPLOYMENT_TO_UPDATE -l ARCHIVE_PATH [-n BLUEPRINT_FILENAME]
 ```
-When updating a deployment using an archive, the name of the blueprint representing the deployment update is assumed to be `blueprint.yaml`. If the blueprint file has a different name, it must be specified using the `-n / --blueprint-filename` optional argument.
+When updating a deployment using an archive, the name of the blueprint representing the deployment update is assumed to be `blueprint.yaml`. If the blueprint file has a different name, it must be specified using the `-n / --blueprint-filename` argument.
 
-{{% gsInfo title="Some Deployment Update Concepts" %}}
-* The **deployment update blueprint** is the blueprint that contains the changes representing the deployment update, what was previously referenced as the 'updated application blueprint'.
```

Comment:
Because the opening 'story' of the doc referenced 'application' in order to be most user friendly.